implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v1/DynatraceMeterRegistryImplV1Test.java

```diff
@@ -40,9 +42,10 @@
  *
  * @author Johnny Lim
  */
-class DynatraceMeterRegistryTest {
+class DynatraceMeterRegistryImplV1Test {
 
     private final DynatraceMeterRegistry meterRegistry = createMeterRegistry();
+    private final DynatraceMeterRegistryImplV1 meterRegistryImpl = createMeterRegistryImpl();
```

Comment:
`meterRegistryImpl` is a weird term, it should be named like `someSpecialMeterRegistry`